media-video/ffmpeg/ffmpeg-4.0.1.recipe

```diff
@@ -123,10 +128,27 @@ CONFLICTS_devel="
 	ffmpeg2${secondaryArchSuffix}_devel
 	"
 
+# -- HACK! --
+# ffmpeg ~0.10 has perpetually had & caused problems when compiled with GCC 2.
+# (Later versions do not compile with GCC 2 at all.) So instead, we now compile
+# FFmpeg for x86_gcc2 with a modern GCC.
+#
+# This works because GCC's C ABI has not changed between 2 and 7; only the C++
```

Comment:
There are no guarantees, and indeed you had to hack some libgcc support.

Reply:
Well, there are no "guarantees", but the C ABI hasn't changed in over 20 years, it probably won't now. :)